samples/bluetooth/direct_test_mode/src/dtm.h

```diff
@@ -15,358 +15,265 @@
-	LE_TRANSMITTER_TEST = 0x2,
-
-	/* Test End Command: End test and send packet report. */
-	LE_TEST_END  = 0x3,
+	/** Bluetooth Low Energy Coded S=2 PHY */
+	DTM_PHY_CODED_S2
 };
 
-/* DTM Test Setup Control codes */
-enum dtm_ctrl_code {
-	/* Reset the packet length upper bits and set the PHY to 1Mbit. */
-	LE_TEST_SETUP_RESET = 0x00,
-
-	/* Set the upper two bits of the length field. */
-	LE_TEST_SETUP_SET_UPPER = 0x01,
-
-	/* Select the PHY to be used for packets. */
-	LE_TEST_SETUP_SET_PHY = 0x02,
-
-	/* Select standard or stable modulation index. Stable modulation index
-	 * is not supported.
-	 */
-	LE_TEST_SETUP_SELECT_MODULATION = 0x03,
-
-	/* Read the supported test case features. */
-	LE_TEST_SETUP_READ_SUPPORTED = 0x04,
-
-	/* Read the max supported time and length for packets. */
-	LE_TEST_SETUP_READ_MAX = 0x05,
-
-	/* Set the Constant Tone Extension info. */
-	LE_TEST_SETUP_CONSTANT_TONE_EXTENSION = 0x06,
-
-	/* Set the Constant Tone Extension slot. */
-	LE_TEST_SETUP_CONSTANT_TONE_EXTENSION_SLOT = 0x07,
+/** @brief DTM modulation index */
+enum dtm_modulation {
+	/** Standard modulation index */
+	DTM_MODULATION_STANDARD,
 
-	/* Set the Antenna number and switch pattern. */
-	LE_TEST_SETUP_ANTENNA_ARRAY = 0x08,
-
-	/* Set the Transmit power. */
-	LE_TEST_SETUP_TRANSMIT_POWER = 0x09
-};
-
-/* DTM Test Setup PHY codes */
-enum dtm_phy_code {
-	/* Set PHY for future packets to use 1MBit PHY.
-	 * Minimum parameter value.
-	 */
-	LE_PHY_1M_MIN_RANGE = 0x04,
-
-	/* Set PHY for future packets to use 1MBit PHY.
-	 * Maximum parameter value.
-	 */
-	LE_PHY_1M_MAX_RANGE = 0x07,
-
-	/* Set PHY for future packets to use 2MBit PHY.
-	 * Minimum parameter value.
-	 */
-	LE_PHY_2M_MIN_RANGE = 0x08,
-
-	/* Set PHY for future packets to use 2MBit PHY.
-	 * Maximum parameter value.
-	 */
-	LE_PHY_2M_MAX_RANGE = 0x0B,
-
-	/* Set PHY for future packets to use coded PHY with S=8.
-	 * Minimum parameter value.
-	 */
-	LE_PHY_LE_CODED_S8_MIN_RANGE = 0x0C,
-
-	/* Set PHY for future packets to use coded PHY with S=8.
-	 * Maximum parameter value.
-	 */
-	LE_PHY_LE_CODED_S8_MAX_RANGE = 0x0F,
-
-	/* Set PHY for future packets to use coded PHY with S=2.
-	 * Minimum parameter value.
-	 */
-	LE_PHY_LE_CODED_S2_MIN_RANGE = 0x10,
-
-	/* Set PHY for future packets to use coded PHY with S=2.
-	 * Maximum parameter value.
-	 */
-	LE_PHY_LE_CODED_S2_MAX_RANGE = 0x13
+	/** Stable modulation index */
+	DTM_MODULATION_STABLE
 };
 
-/* DTM Test Setup Read supported parameters codes. */
-enum dtm_read_supported_code {
-	/* Read maximum supported Tx Octets. Minimum parameter value. */
-	LE_TEST_SUPPORTED_TX_OCTETS_MIN_RANGE = 0x00,
-
-	/* Read maximum supported Tx Octets. Maximum parameter value. */
-	LE_TEST_SUPPORTED_TX_OCTETS_MAX_RANGE = 0x03,
-
-	/* Read maximum supported Tx Time. Minimum parameter value. */
-	LE_TEST_SUPPORTED_TX_TIME_MIN_RANGE = 0x04,
+/** @brief DTM maximum supported values */
+enum dtm_max_supported {
+	/** Max supported TX octets */
```

Comment:
```suggestion
	/** Maximum supported TX octets. */
```
Please use full forms instead of **max** and **min** - please correct all occurrences in this file. Please also add periods at the end of all doxygen comments.